share/spice/sports/mlb/games/sports_mlb_games.js

```diff
@@ -0,0 +1,175 @@
+(function (env) {
+    "use strict";
+
+    env.ddg_spice_sports_mlb_games = function(apiResult) {
+
+        if (!apiResult || !apiResult.data || !apiResult.data.games || !apiResult.data.games.length) {
+            return Spice.failed('mlb_games');
+        }
+        
+        DDG.require('moment.js', function(){
+            Spice.add({
+                id: 'mlb_games',
+                name: 'MLB Games',
+
+                data: gameData(apiResult.data),
+
+                from: apiResult.from,
+                signal: apiResult.signal,
+
+                meta: {
+                    idField: 'id',
+                    showMoreAt: true,
+                    sourceIcon: false,
+                    sourceUrl: "http://www.bleacherreport.com/",
```

Comment:
It would be nice if we could link to the team page for the team they searched for, but I don't see an easy way to do it given the current api response. Maybe we should at least link to the league page? i.e. `http://www.bleacherreport.com/mlb`?